_data/vnext_office_hours.yml

```diff
@@ -22,5 +22,5 @@ extenders:
   - date: 10/06/2022 8AM - 9AM ET
```

Comment:
Should not be 10/06/2023?

Reply:
Tahnks for catching. Also fixed for other office hours.